ara/callback/log_ara.py

```diff
@@ -0,0 +1,226 @@
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+import logging
+import os
+from datetime import datetime
+from decorator import decorator
+
+from ara import app, db, models
+from ansible.plugins.callback import CallbackBase
+try:
+    import simplejson as json
+except ImportError:
+    import json
+
+__metaclass__ = type
+
+LOG = logging.getLogger(__name__)
+
+# Set ARA_DEBUG_LOG to a path if you want the callback plugin to
+# log debug messages to a file.
+if os.getenv('ARA_DEBUG_LOG'):
```

Comment:
Maybe.  I was mostly considering this  a developer option; I thought about ripping out all the logging before submitting the PR, but I'll look at rolling that into the normal config process.